.github/CONTRIBUTING.md

```diff
@@ -48,6 +48,10 @@ In addition to writing properly formatted commit messages, it's important to inc
 
 For package version upgrades and such a one-line commit message is usually sufficient.
 
+## Backporting changes
+
+To backport a change into a release channel, take note of the commit in which the change was introduced into `master`. Then check out the release channel and use `git cherry-pick -x <original commit>`. When opening your backport PR, make sure to select the release channel branch as the target branch of the PR, and link to the PR in which the original change was made to `master`.
```

Comment:
Perhaps also clarify what the release branches look like, pull requests should target eg. `release-19.09` and not the channel (`nixos-19.09`/`nixpkgs-unstable`) branches directly.

Reply:
Also see https://nixos.org/nixpkgs/manual/#submitting-changes-stable-release-branches.

Reply:
Thanks! I've added a link and an example.